### ISSUE TITLE
Possible closure_end_indentation false positive

### ISSUE THREAD

**Issue by keith** [realm/SwiftLint#1216]
```swift
foo(foo: bar,
    options: baz) { _ in }
```

I'm not sure if we want to handle this case, but this is "valid" in our codebase when the first line of the call exceeds our line length limit, but triggers a warning with the `closure_end_indentation` rule.

// @marcelofabri

**Comment by marcelofabri** [comment]
I think this should be valid.

**Comment by keith** [comment]
Another case:

```swift
foo(abc, 123)
{ _ in }
```
